airflow/gcp/hooks/cloud_storage_transfer_service.py

```diff
@@ -370,7 +370,7 @@ def resume_transfer_operation(self, operation_name: str):
     def wait_for_transfer_job(
         self,
         job: Dict,
-        expected_statuses: Tuple[str] = (GcpTransferOperationStatus.SUCCESS,),
+        expected_statuses: Optional[Set[str]] = None,
```

Comment:
@mik-laj WDYT? Should we use `Optional` or not?

Reply:
I prefer less code.

Reply:
Agree 🐈